doc/lua_api.txt

```diff
@@ -2921,7 +2921,8 @@ Definition tables
         collisionbox = {-0.5,-0.5,-0.5, 0.5,0.5,0.5},
         visual = "cube"/"sprite"/"upright_sprite"/"mesh"/"wielditem",
         visual_size = {x=1, y=1},
-        mesh = "model",
+        mesh = "model", -- for mobs (0, 0, 0) is ground level,
+			-- for players (0, -1, 0) is ground level.
```

Comment:
are these really the only two categories? It should be:

    -- for players (0, -1, 0) is ground level,
    -- for all other entities, (0, 0, 0) is ground level.